### PR TITLE
Add missing thanks

### PR DESCRIPTION
**Pull request by NavidZ**
@patrickhlauke is there anyone else missing here from the list?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -991,14 +991,18 @@ <h1>Acknowledgments</h1>
             Cathy Chan,
             Ted Dinklocker,
             Dave Fleck,
+            Ella Ge,
             Scott González,
+            Philippe Le Hegaret,
```
Comment (patrickhlauke) [inline index.html]: "Philippe Le Hégaret" (missing the accent)

**Comment by patrickhlauke** [comment]
Looking over our issues/prs, I'd add: Patrick Kettner, Scott Low, Ming-Chou Shih, Brenton Simpson

File: index.html (hunk 1 above)
Comment (NavidZ) [reply index.html]: Thanks. Done. I have added others as well.

**Comment by patrickhlauke** [comment]
:+1:

**Pull request merged**